.github/pull_request_template.md

```diff
@@ -28,6 +28,7 @@ Fixes #
 - [ ] Style
 - [ ] Unit tests
 - [ ] Subsystem tests
+- [ ] e2e tests
```

Comment:
There is `CI` option. Is there any difference with  `e2e tests`?

Reply:
`e2e tests` would be the tests cases / utils code update and `CI` would be the ci folder ?

Reply:
I don't think the items indicates the folder. They should be for the updating purpose. All QE's updating can be aligned to CI. @nirarg Can you please help to confirm that?

Reply:
cc @gdbranco @ciaranRoche

Reply:
+1 on QE updating to be aligned to CI

Reply:
ok then, will setup in `ci`